product_docs/docs/pem/8/monitoring_BDR_nodes.mdx

```diff
@@ -1,25 +1,25 @@
 ---
-title: "Monitoring BDR"
-# This file is moved for pem_ent_feat/17_monitoring_BDR_nodes.mdx
+title: "Monitoring PGD"
+# This file is moved for pem_ent_feat/17_monitoring_PGD_nodes.mdx
 ---
 
 !!! Tip "New Feature "
-    BDR support is available in PEM version 8.1.0 and later.
+    PGD support is available in PEM version 8.1.0 and later.
 
-BDR (Bi-Directional Replication) is a PostgreSQL extension that provides multi-master replication between the Postgres clusters. Before you monitor BDR through PEM console, you must ensure that your BDR nodes are up and running. For more information, see [BDR](/bdr/latest).
+PGD (Bi-Directional Replication) is a PostgreSQL extension that provides multimaster replication between the Postgres clusters. Before you monitor PGD through PEM console, you must ensure that your PGD nodes are up and running. For more information, see [PGD](/pgd/latest).
```

Comment:
Is this what PGD stands for? I think you want the full name written out first and then the initialism.